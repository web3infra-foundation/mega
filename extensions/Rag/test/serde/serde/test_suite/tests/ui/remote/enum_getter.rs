use serde_derive::Serialize;

mod remote {
    pub enum E {
        A { a: u8 },
    }
}

#[derive(Serialize)]
#[serde(remote = "remote::E")]
pub enum E {
    A {
        #[serde(getter = "get_a")]
        a: u8,
    },
}

fn main() {}
