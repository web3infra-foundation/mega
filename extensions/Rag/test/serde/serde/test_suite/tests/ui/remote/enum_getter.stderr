error: #[serde(getter = "...")] is not allowed in an enum
  --> tests/ui/remote/enum_getter.rs:10:1
   |
10 | / #[serde(remote = "remote::E")]
11 | | pub enum E {
12 | |     A {
13 | |         #[serde(getter = "get_a")]
14 | |         a: u8,
15 | |     },
16 | | }
   | |_^
